travis/samples/ngx-couchbase/.yo-rc.json

```diff
@@ -1,11 +1,11 @@
 {
     "generator-jhipster": {
-      "baseName": "travisCouchbaseInfinispan",
+      "baseName": "travisCouchbaseNoCache",
       "packageName": "io.github.jhipster.travis",
       "packageFolder": "io/github/jhipster/travis",
       "serverPort": "8080",
       "authenticationType": "jwt",
-      "cacheProvider": "infinispan",
+      "cacheProvider": "no",
```

Comment:
I guess it wouldnt hurt to have any cache here to see if things compile when cache is used with noSQL